index.js

```diff
@@ -408,32 +397,36 @@ class McEncryptionSchemePolyfill {
    * @return {!MediaKeySystemConfiguration}
    */
   static convertToMediaKeySystemConfig_(decodingConfig) {
-    const mediaCapkeySystemConfig = decodingConfig.keySystemConfiguration;
+    const mediaCapKeySystemConfig = decodingConfig.keySystemConfiguration;
     const audioCapabilities = [];
     const videoCapabilities = [];
 
-    if (mediaCapkeySystemConfig.audio) {
+    if (mediaCapKeySystemConfig.audio) {
       const capability = {
-        robustness: mediaCapkeySystemConfig.audio.robustness || '',
+        robustness: mediaCapKeySystemConfig.audio.robustness || '',
         contentType: decodingConfig.audio.contentType,
       };
       audioCapabilities.push(capability);
     }
 
-    if (mediaCapkeySystemConfig.video) {
+    if (mediaCapKeySystemConfig.video) {
       const capability = {
-        robustness: mediaCapkeySystemConfig.video.robustness || '',
+        robustness: mediaCapKeySystemConfig.video.robustness || '',
         contentType: decodingConfig.video.contentType,
       };
       videoCapabilities.push(capability);
     }
 
+    const mediaCapInitDataType = mediaCapKeySystemConfig.initDataType;
+    const initDataTypes = mediaCapInitDataType && mediaCapInitDataType.length :
```

Comment:
Just a boolean check should suffice.  If it's a non-empty string, it will be truthy.